openprescribing/measure_definitions/opioidper1000.json

```diff
@@ -4,34 +4,34 @@
     "high dose opioid items per 1000 patients"
   ],
   "description": [
-    "Opioids with likely daily dose of ≥120mg morphine equivalence per 1000 patients"
+    "Opioids with likely daily dose of \u2265120mg morphine equivalence per 1000 patients"
   ],
+  "numerator_short": "High dose opioid items",
+  "denominator_short": "1000 patients",
   "why_it_matters": [
     "The <a href='https://www.rcoa.ac.uk/faculty-of-pain-medicine/opioids-aware'> Opioids Aware</a> project seeks to ",
     "improve prescribing of opioid analgesia.  There is little evidence that opioids are helpful in long term pain, ",
     "and the risk of harm increases significantly above 120mg morphine (or equivalent) per day, without much increase in benefit. ",
     "We have assumed that if a patient is on regular doses of 120mg morphine a day or above that they are likely to have also had ",
-    "additional opioids for breakthrough pain.  This is why we have set the threshold at ≥120mg morphine equivalence per day. ",
+    "additional opioids for breakthrough pain.  This is why we have set the threshold at \u2265120mg morphine equivalence per day. ",
```

Comment:
I think spec-compliant JSON is supposed to be ascii only. Although I'm pretty sure Python's parser will happily take UTF-8 encoded input.

Reply:
I just looked this up cos I was interested

> JSON text SHALL be encoded in UTF-8, UTF-16, or UTF-32.  The default encoding is UTF-8

https://tools.ietf.org/html/rfc7159

Reply:
Oh good! I'm wondering what the source of my false memory is now. It might be something to do with the fact that it _can_ be ascii and possibly certain browsers tripping up on certain unicode characters. But anyway, glad to be corrected.

Reply:
> Do we really want unicode literals?

No, well spotted.  These must've snuck in when I rewrote all the measure definitions using `json.dump()`.  Will fix.